vividus-plugin-rest-api/src/main/java/org/vividus/archive/steps/ArchiveSteps.java

```diff
@@ -38,11 +40,15 @@
 {
     private final ISoftAssert softAssert;
     private final VariableContext variableContext;
+    private final IAttachmentPublisher attachmentPublisher;
+    private boolean attachmentLoaded;
```

Comment:
ArchiveSteps bean is shared across all running threads so it should not have state at the class level

Reply:
fixed, thank you